render/egl.c

```diff
@@ -19,7 +19,11 @@ static bool egl_get_config(EGLDisplay disp, EGLint *attribs, EGLConfig *out,
 		return false;
 	}
 
-	EGLConfig configs[count];
+	EGLConfig *configs = malloc(sizeof(EGLConfig) * count);
+	if (configs == NULL) {
+		wlr_log(WLR_ERROR, "Failed to allocate memory");
+		return false;
+	}
 
 	ret = eglChooseConfig(disp, attribs, configs, count, &matched);
 	if (ret == EGL_FALSE) {
```

Comment:
(insert same fuck github comment) -- one return false below this line

Reply:
I'm not sure you're referring to here, probably because of the issue you pointed out with GitHub's comments. Perhaps you could copy the code context into a comment.

Reply:
Litteraly two lines below is a return false leaking configs:

```c
	ret = eglChooseConfig(disp, attribs, configs, count, &matched);
	if (ret == EGL_FALSE) {
		wlr_log(WLR_ERROR, "eglChooseConfig failed");
		return false;
	}
```